### PR TITLE
ssh-copy-id: move to common

### PR DESCRIPTION
**Pull request by josa42**

### CODE REVIEW

File: pages/common/ssh-copy-id.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# ssh-copy-id
+
+> Install your public key in a remote machine's authorized_keys.
+
+- Copy your key to a remot machine:
```
Comment (waldyrious) [inline pages/common/ssh-copy-id.md]: What's the default if there are several local keys available? Also, typo: "remot" --> "remote".

File: pages/common/ssh-copy-id.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+`ssh-copy-id {{user@remote}}`
+
+- Copy a specific key to a remot machine:
+
+`ssh-copy-id -i {{~/.ssh/id_example.pub}} {{user@remote}}`
```
Comment (waldyrious) [inline pages/common/ssh-copy-id.md]: Do you think it'd be worth changing the token to "remote_host", for consistency with [ssh.md](https://github.com/tldr-pages/tldr/blob/master/pages/common/ssh.md), [scp.md](https://github.com/tldr-pages/tldr/blob/master/pages/common/scp.md), and probably others?

**Comment by waldyrious** [comment]
Thanks for the PR! Please check the inline comments.

**Comment by agnivade** [comment]
Nice PR! Very useful command. I keep telling people to stop copying their public key and manually adding to `authorized_keys`.

File: pages/common/ssh-copy-id.md [hunk 3]
```diff
@@ -2,6 +2,14 @@
 
 > Install your public key in a remote machine's authorized_keys.
 
+- Copy your keys to the remote machine:
+
+`ssh-copy-id {{username@remote_host}}`
+
+- Copy your default key to the remote machine:
```
Comment (agnivade) [inline pages/common/ssh-copy-id.md]: This is not useful IMO. Copying your default key with ssh would require access to the machine in the first place. Which means you have to give password auth or some other key.

**Comment by agnivade** [comment]
@josa42 - This was a long time pending. So just changed it myself. Let me know if you have any other thoughts.

**Pull request merged**